src/sdk/src/session_sdk.rs

```diff
@@ -0,0 +1,133 @@
+//
+// mutation_sdk.rs
+// Copyright (C) 2022 db3.network Author imotai <codego.me@gmail.com>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use chrono::Utc;
+
+#[derive(Debug, PartialEq)]
+pub enum SessionStatus {
+    READY,
+    RUNNING,
+    BLOCKED,
+}
+#[derive(Debug)]
+pub struct SessionManager {
+    id: i32,
+    start_time: i64,
+    query_count: i32,
+    status: SessionStatus,
+}
+// default session timeout 60s
+pub const DEFAULT_SESSION_PERIOD: i64 = 60;
+// default session limit
+pub const DEFAULT_SESSION_QUERY_LIMIT: i32 = 100;
+
```

Comment:
I think both SDK and the validator will use session manager and it's not good put it in SDK

Reply:
will fix in the next revision

Reply:
Resolved in PR https://github.com/dbpunk-labs/db3/pull/146